lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/AllocatedEvaluator.java

```diff
@@ -96,4 +96,16 @@ void submitContextAndTask(final Configuration contextConfiguration,
   void submitContextAndServiceAndTask(final Configuration contextConfiguration,
                                       final Configuration serviceConfiguration,
                                       final Configuration taskConfiguration);
+
+  /**
+   * Check whether this evaluator is in closing state.
+   * @return whether this evaluator is in closing state.
+   */
+  boolean isClosing();
+
+  /**
+   * Check whether this evaluator is in closed state.
+   * @return whether this evaluator is in closed state.
+   */
+  boolean isClosed();
```

Comment:
Oh, I should have asked this!
These functions are called during test.
I couldn't find any way to test w/o these methods. 
Do you think that this is okay or have better idea?